src/main/groovy/org/transmartproject/core/dataquery/highdim/tworegion/EventValues.groovy

```diff
@@ -0,0 +1,10 @@
+package org.transmartproject.core.dataquery.highdim.tworegion
+
+/**
+ * Represents metadata about one event, possibly linking to multiple junctions
+ */
+interface EventValues {
+    String getCgaType();
+    String getSoapClass();
+    JunctionInEventValues[] getJunctions();
```

Comment:
Because the type you return conveys information to the client. A list indicates that there may be repetitions and that the order matters. An array is even worse because it presents less abstraction than a list (not to mention it's less convenient for the caller).